docs/sourcegraph-accounts/index.mdx

```diff
@@ -24,6 +24,9 @@ Users who had attempted to create an account but never verified their primary em
 1. Visit https://accounts.sourcegraph.com/reset-password to complete the password reset.
 1. Upon successful password reset, the current session is automatically signed in.
 
+### I am currently using a social sign-in, can I switch to email/password?
+At the moment changing to email/password login is not possible. If for whatever reason you lose access to your social sign-in please contact Support for further assistance.
```

Comment:
It should/is actually possible by following the Q right above this one 🤔 

![CleanShot 2024-06-17 at 18 09 10@2x](https://github.com/sourcegraph/docs/assets/2946214/ec4e38e7-7278-48c5-8f38-5788d00d9085)

Reply:
I thought this wasn't allowed 🤔 

That makes sense though... I'll close this PR then and start pointing users to this!